daemon/cmd/kube_proxy_replacement.go

```diff
@@ -448,6 +415,43 @@ func finishKubeProxyReplacementInit(isKubeProxyReplacementStrict bool) {
 
 	// After this point, BPF NodePort should not be disabled
 
```

Comment:
I think we need this as separate fix for 1.9 as well?

Reply:
Yeah, I have it in my TODO.